### PR TITLE
Add release notes for triggers from 0.11.0 to 0.12.1

### PR DESCRIPTION
**Pull request by savitaashture**

### CODE REVIEW

File: openshift/release/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+    Change the event ID representation from a 5 character random string to a UUID. 
+
+
+## Fixes :bug:
+
+* Send Tekton installation namespace to EL ([#927](https://github.com/tektoncd/triggers/pull/927))
+* Add validation to the length of EventListener name ([#935](https://github.com/tektoncd/triggers/pull/935))
+* Use generateName instead of $(uid) for metadata.name ([#939](https://github.com/tektoncd/triggers/pull/939))
+* Fix container name in Deployment for interceptors ([#956](https://github.com/tektoncd/triggers/pull/956))
```
Comment (savitaashture) [inline openshift/release/release-notes.md:67]: can we remove this entire fixes ?

File: openshift/release/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+* Add links to versioned docs for v0.11.1 (#918)
+* Remove deprecated field template.Name in favour of template.Ref (#919)
+* Add links to versioned docs for v0.11.2 (#930)
+* Make test names consistent in reconciler (#937)
+* K8s dependency bump to v0.19.7(#934)
+* Add kustomize.yaml to the Tekton folder (#950)
+* Adding tests for cel expressions demonstrating use of has (#947)
+* Use tekton beta resources for build (#957)
+* Use golang-build-legacy and golang-test-legacy. (#965)
```
Comment (savitaashture) [inline openshift/release/release-notes.md:80]: same here
we can remove

File: openshift/release/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+## Docs :book:
+
+* Update min K8s version for Triggers (#922)
+* Add link to godoc reference (#933)
+* Add some documentation around debugging EventListeners. (#954)
+* doc: template inside a Trigger is not optional (#963)
+* Rewrite the Tekton Triggers installation instructions for clarity and flow (#966)
+* Write a new Tekton Triggers Overview (#972)
+* Update to match the new /docs/README.md Triggers Overview from PR 972 (#975)
```
Comment (savitaashture) [inline openshift/release/release-notes.md:90]: this as well

File: openshift/release/release-notes.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+This PR fixes by having the Reconciler send the installation namespace as a environment variable set on the EventListener's pod.
+
+# Tekton Triggers v0.11.1
+
+## Deprecation Notices :rotating_light:
+
+* Revert removal of template.Name field and handle Deprecation of template.Name field ([#912](https://github.com/tektoncd/triggers/pull/912))
+
+  Deprecate template.Name in favor of `template.Ref`
+
```
Comment (savitaashture) [inline openshift/release/release-notes.md:108]: I think we can completely remove this part cc @khrm

File: openshift/release/release-notes.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+* Merge extensions into body for webhook interceptors ([#859](https://github.com/tektoncd/triggers/pull/859))
+  
+  Extensions added by a CEL Interceptor will be passed on to webhook interceptors by merging the extension fields into the event body under a `extensions` field. See [docs/eventlisteners.md](https://github.com/tektoncd/triggers/blob/master/docs/eventlisteners.md#chaining-interceptors) for more details.
+* Bind admin role so that SA can use PSP ([#863](https://github.com/tektoncd/triggers/pull/863))  
+* Update tekton eventlistener portname owing to istio naming conventions ([#880](https://github.com/tektoncd/triggers/pull/880))
+  
+  Event listener port should be called `http-listener` instead of `listener`.
+* Update TLS example ([#885](https://github.com/tektoncd/triggers/pull/885))
+* Use system.DefaultNamespace for interceptor urls ([#891](https://github.com/tektoncd/triggers/pull/891))
+* Add core interceptors image to created images ([#895](https://github.com/tektoncd/triggers/pull/895))
```
Comment (savitaashture) [inline openshift/release/release-notes.md:156]: can we remove this fixes also cc @khrm

File: openshift/release/release-notes.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+* Fix behaviour which allows EL ports to be configured ([#881](https://github.com/tektoncd/triggers/pull/881))  
+* Refactor sink test ([#882](https://github.com/tektoncd/triggers/pull/882))
+* Update service account name for TLS example ([#884](https://github.com/tektoncd/triggers/pull/884))  
+* Add https-listener service port name ([#887](https://github.com/tektoncd/triggers/pull/887))
+* Improve the TriggerTemplate message ([#899](https://github.com/tektoncd/triggers/pull/899))
+  
+  Improve the error message when parsing the results of rendering a TriggerTemplate.
+* Bump knative to release-0.20 and pipeline to v0.20.0 ([#903](https://github.com/tektoncd/triggers/pull/903))
+
+**This bumps the minimum supported version of kubernetes to 0.17**
```
Comment (savitaashture) [inline openshift/release/release-notes.md:176]: same here

File: openshift/release/release-notes.md [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+
+## Docs :book:
+* Add links to versioned docs for v0.10.1 ([#851](https://github.com/tektoncd/triggers/pull/851))
+* Remove mention of dep in dev docs ([#866](https://github.com/tektoncd/triggers/pull/866))
+* Add docs for installing nightly release ([#874](https://github.com/tektoncd/triggers/pull/874))
+* Fixed link in EventListener secure connection docs ([#905](https://github.com/tektoncd/triggers/pull/905))
+
+  Link fixed in the docs for the EventListener example with secure connection.
+* Add links to versioned docs for v0.10.2 ([#907](https://github.com/tektoncd/triggers/pull/907))
```
Comment (savitaashture) [inline openshift/release/release-notes.md:185]: this as well

**Comment by savitaashture** [comment]
@khrm @vdemeester @piyush-garg 

I just added comments to remove part of the release notes if its ok then i can remove those and update PR

**Comment by savitaashture** [comment]
> @khrm @vdemeester @piyush-garg
> 
> I just added comments to remove part of the release notes if its ok then i can remove those and update PR

I have removed all those which are not required

**Review by vdemeester — APPROVED** [review]
/lgtm

**Pull request merged**